src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -29,18 +31,21 @@ internal class BlobsExtensionConfigProvider : IExtensionConfigProvider,
         private IContextGetter<IBlobWrittenWatcher> _blobWrittenWatcherGetter;
         private readonly INameResolver _nameResolver;
         private IConverterManager _converterManager;
+        private readonly ILogger _logger;
```

Comment:
Let's name this _bindingStatsLogger so people don't accidentally use it for general logging